src/vivarium/framework/lookup/manager.py

```diff
@@ -52,7 +51,7 @@ def name(self) -> str:
         return "lookup_table_manager"
 
     def setup(self, builder: "Builder") -> None:
-        self.tables = {}
+        self.tables: dict[int, LookupTable] = {}
```

Comment:
I would have guessed this is a str and not int with the way we set up lookup tables. Where do we index them with ints?

Reply:
ln 127